api/identity.ts

```diff
@@ -15,11 +14,11 @@ export interface Identity {
 }
     
 export class IdentityService {
-    constructor() {
-        this._store = new sm.Store();
+    constructor(private store: IStore) {
+        this._store = store;
```

Comment:
a repository abstraction is being used, but by referencing a concrete implementation explicitly (a store, directly referencing nedb) in its constructor theres tight coupling between business domain and the persistence layer.

injecting a store-like instance allows the persistence layer to be stubbed out for testability / flexibility. can change the store w/o modifying the service.